### ISSUE TITLE
RCS1047 is reported for ValueTask

### ISSUE THREAD

**Issue by aelij** [JosefPihrt/Roslynator#320]
For example:

```csharp
ValueTask<int> DoAsync()
```

**Comment by JosefPihrt** [comment]
This issue has been fixed in #261. Are you using latest version?

**Comment by aelij** [comment]
v1.7.0

**Comment by JosefPihrt** [comment]
Latest version is 1.8.0.

If you are using package Roslynator.Analyzers you can install 1.8.0-rc or wait a couple of days before 1.8.0 is released.

**Comment by thomaslevesque** [comment]
It works fine for `ValueTask<int>`, but it still incorrectly reports RCS1047 for `ValueTask`.
